packages/app/src/hooks/useIpfs.ts

```diff
@@ -4,7 +4,7 @@ import axios from "axios"
 import { useNotification } from "./useNotification"
 import { getClient } from "../services/ipfs"
 
-const IPFS_GATEWAY = process.env.REACT_APP_IPFS_GATEWAY
+export const IPFS_GATEWAY = process.env.REACT_APP_IPFS_GATEWAY
```

Comment:
```suggestion
const IPFS_GATEWAY = process.env.REACT_APP_IPFS_GATEWAY
```
This should not be exported and never used directly. 
It's a fallback that will only be used if no IPFS node is provided by the user. 

We should strive to use the one provided by the user via the useIpfs hook. If, however, we require a URL for, for instance, getting an image, we should use the exported wrapper functions instead, like `getText` and `getImgSrc`, or create a new one if required.

FYI: `getImgSrc` is right now using the `IPFS_GATEWAY`, but this is a provisional workaround that will be fixed, so we should still use the `getImgSrc`.